go.mod

```diff
@@ -0,0 +1,39 @@
+module github.com/instaclustr/cassandra-operator
+
+require (
+	github.com/NYTimes/gziphandler v1.0.1 // indirect
+	github.com/go-logr/logr v0.1.0
+	github.com/go-openapi/spec v0.19.0
+	github.com/google/uuid v1.1.1
+	github.com/operator-framework/operator-sdk v0.10.1-0.20190809033248-9d6ffddcd86f
+	github.com/spf13/pflag v1.0.3
+	gopkg.in/resty.v1 v1.12.0
+	gopkg.in/yaml.v2 v2.2.2
+	gotest.tools v2.2.0+incompatible
+	k8s.io/api v0.0.0-20190612125737-db0771252981
+	k8s.io/apimachinery v0.0.0-20190612125636-6a5db36e93ad
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/kube-openapi v0.0.0-20190603182131-db7b694dc208
+	sigs.k8s.io/controller-runtime v0.1.12
+	sigs.k8s.io/controller-tools v0.1.10
+)
+
+// Pinned to kubernetes-1.13.4
```

Comment:
@johananl why this version? There is 1.15 already, is operator-sdk 0.10.0 known to work with this one?

Reply:
Good question. This file is generated by the SDK. I'll have to check this.